README.md

```diff
@@ -33,18 +33,25 @@ If you can live with latency, bugged detections and want to have data privacy, f
         position: 'bottom_bar',
         config: {
             microphone: 1,
+						mode:  see below,
             ...
+						
         }
     }
     ```
 
 ## Config Options
 
-| **Option** | **Default** | **Description** |
+| **Option** | **Default** | **Description** | |
 | --- | --- | --- |
 | `microphone` | REQUIRED | Id of microphone shown in the installer. |
 | `keyword` | `'MAGIC MIRROR'` | Keyword the mirror starts to listen. IMPORTANT: Only UPPERCASE Letters |
 | `timeout` | `15` | time the keyword should be active without saying something |
+| `mode` | OPTIONAL | |
```

Comment:
this block doesn't fit the schema of this table (one option on line), please adjust or if you think it will be to crowded, then extract it in a seperate table

Reply:
changed